packages/ui/client/components/views/ViewConsoleOutputEntry.cy.tsx

```diff
@@ -0,0 +1,42 @@
+import Filter from 'ansi-to-html'
+import ViewConsoleOutputEntry from './ViewConsoleOutputEntry.vue'
+
+const htmlSelector = '[data-type=html]'
+const textSelector = '[data-type=text]'
+
+describe('ViewConsoleOutputEntry', () => {
+  it('test plain entry', () => {
+    const content = new Date().toISOString()
+    const container = cy.mount(
+      <ViewConsoleOutputEntry
+        task-name="test/text"
+        type="stderr"
+        time={Date.now()}
+        html={false}
+        content={content}
+      />,
+    ).get(textSelector)
+    container.should('exist')
+    container.invoke('text').then((t) => {
+      expect(t, 'the message has the correct message').equals(content)
+    })
+  })
+  it('test html entry', () => {
+    const now = new Date().toISOString()
+    const content = new Filter().toHtml(`\x1B[33m${now}\x1B[0m`)
+    const container = cy.mount(
+      <ViewConsoleOutputEntry
+        task-name="test/html"
+        type="stderr"
+        time={Date.now()}
+        html={true}
+        content={content}
+      />,
+    ).get(htmlSelector)
+    container.should('exist')
+    container.children('span').then((c) => {
```

Comment:
Instead of using DOM tags like `span` and hierarchy like `children`, just as the test for what you want -- you want to find the element with the text `now`.

This can be done via

```js
cy.contains(now)
```

and you can delete all references to span, etc.

the `contains` check **implicitly checks that `now` exists in the DOM** so you ca delete `container.should('exist')`.

`cy.contains` is very similar to Testing Library's `findByText` and I may suggest installing `@testing-library/cypress` to have similar bindings throughout the codebase.